src/dcmp/dcmp.c

```diff
@@ -582,10 +580,6 @@ static int dcmp_compare_data(
         if (src_read != dst_read) {
             /* one read came up shorter than the other */
             rc = 1;
-            copy_src_to_dst = 1;
-            if (!mfu_copy_opts->do_sync) { 
```

Comment:
want to keep this break?

Reply:
@adammoody yeah put it back. Deleted it by accident. Thanks.